### DISCUSSION TITLE
Arabic and Dialects

### DISCUSSION THREAD

**Discussion by hasbean** [openai/whisper#466] [General]
Hello and thanks for this incredible piece of awesome.

I've tested it out on actual "traditional" Arabic and it seems to work great with Large.  However, the AI seems to have no understanding of Arabic dialects whatsoever.

Outside of news channels and some historic flicks, no one actually speaks traditional Arabic.  Every Arabic country has its own spin on the language to the point where some dialects aren't even understood by Arabic speakers.  To make matters worse, some speakers may switch dialects mid conversation just to make their point clear to the listener.

Would love to see whisper include the ability to figure out dialects in conversations.

**Comment by Keysz** [comment]
I couldn't agree more. Would like to see capabilities for dialects developed

**Comment by ahmed-adly-khalil** [comment]
+1 totally agree, actually i think no Arabic speaker would disagree ...

**Comment by mshparber** [comment]
Strange, since the regular ChatGPT 4o is able to understand spoken Levantine dialect. It also is able to speak out load spoken Levantine. 
How do they bring this to Wisper?

**Comment by darkzbaron** [comment]
+1

**Comment by ramsha-g** [comment]
I'm working on it, starting with Emirati

**Comment by mawnir** [comment]
Any updates on this?
